src/provider/data.rs

```diff
@@ -1286,6 +1315,25 @@ static P_YANDEX_RU: Lazy<Provider> = Lazy::new(|| Provider {
     oauth2_authorizer: Some(Oauth2Authorizer::Yandex),
 });
 
+// yggmail.md: yggmail
+static P_YGGMAIL: Lazy<Provider> = Lazy::new(|| {
+    Provider {
+    id: "yggmail",
+    status: Status::Preparation,
+    before_login_hint: "An Yggmail companion app needs to be installed on your device to access the Yggmail network.",
+    after_login_hint: "Make sure, the Yggmail companion app runs whenever you want to use this account. Note, that you usually cannot write from @yggmail addresses to normal e-mail-addresses (as @gmx.net). However, you can create another account in the normal e-mail-network for this purpose.",
+    overview_page: "https://providers.delta.chat/yggmail",
+    server: vec![
+        Server { protocol: Imap, socket: Plain, hostname: "localhost", port: 1143, username_pattern: Email },
+        Server { protocol: Smtp, socket: Plain, hostname: "localhost", port: 1025, username_pattern: Email },
+    ],
+    config_defaults: None,
+    strict_tls: true,
```

Comment:
oh, wondering if that really works. maybe better  disable `strict_tls` - it is probably not needed anyway as the connection is only local on the same device.

Reply:
i did a pr for that https://github.com/deltachat/provider-db/pull/201

Reply:
this is okay, see comments in https://github.com/deltachat/provider-db/pull/201 and https://github.com/deltachat/provider-db/pull/202